test/lib/cacheWrapperTest.js

```diff
@@ -1,128 +1,465 @@
 'use strict';
 
-process.env.NODE_ENV = 'development';
-
-var cache = require( '../../lib/cacheWrapper' );
-var chai = require( 'chai' );
-chai.use( require( 'chai-as-promised' ) );
-var expect = chai.expect;
-
-var Catbox = require( 'catbox' );
-
-var sinon = require( 'sinon' );
-
-var policies = [ {
-  segment: 'foo',
-  expiresIn: 10000
-} ];
-
-var serverConfig = {};
-
-var cacheItem = {
-  segment: 'foo',
-  key: 'bar',
-  value: 'baz'
-};
-
-var nonExistantCacheItem = {
-  segment: 'foo',
-  key: 'noExist'
+var sinon = require('sinon');
+var sandbox = sinon.sandbox.create();
+var expect = require('chai')
+  .use(require('dirty-chai'))
+  .use(require('chai-as-promised'))
+  .use(require('sinon-chai'))
+  .expect;
+var redisWrapper = require('../../lib/redisWrapper');
+
+var Q = require('q');
+var rewire = require('rewire');
```

Comment:
I used rewire as was trying to mock some private vars and functions but will raise an issue so it can be replaced by a better approach